worker/wallet.js

```diff
@@ -161,6 +161,10 @@ export async function checkInvoice ({ data: { hash, invoice }, boss, models, lnd
   }
 }
 
+export async function retryTimeout ({ data: { hash }, models, lnd, boss }) {
+  await models.invoice.update({ where: { hash }, data: { updatedAt: new Date() } })
```

Comment:
This might need to set a date slightly in the future to avoid race conditions with the notification indicator query but not sure, need to test

Reply:
This fixes the notification indicator, but it doesn't seem to work as expected: It does not seem to use the date passed to `updatedAt`. I suspect the trigger for updated_at overrides any value passed to `updatedAt`, however, the value set does not seem to be the actual update date :thinking: 

Or maybe this does not run when I think it does run.

Reply:
Mhh, it actually works as expected. It DOES use the `updatedAt` I passed in. Not sure why I was confused before. Updated to use a date 5 seconds in the future just to be sure.